internal/provider/data_source_packer_image_iteration.go

```diff
@@ -175,6 +181,11 @@ func dataSourcePackerImageIterationRead(ctx context.Context, d *schema.ResourceD
 	if err := d.Set("created_at", iteration.CreatedAt.String()); err != nil {
 		return diag.FromErr(err)
 	}
+	if !time.Time(iteration.RevokeAt).IsZero() {
```

Comment:
`iteration.RevokeAt` is a value returned in API response. In this PR I'm making it available in the data source output. I do the zero check here, before writing it in the output, because the default value for strfmt.DateTime{} is `0001-01-01T00:00:00.000Z`, so to avoid writing the zero value in the output.

Reply:
Ooh I misread this haha :doh: